code/index.html

```diff
@@ -5,33 +5,82 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <title>Cluedo</title>
+    <link rel="shortcut icon" href="./assets/splash.png" type="image/png" />
   </head>
   <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
+    <!-- HEADER -->
+    <header class="header-wrapper visible" id="startWindow">
+      <div class="header-container">
+        <h1 class="header-text">CLUEDO</h1>
+        <div class="drop"></div>
+      </div>
+      <button class="button button-start" id="startButton">LET'S PLAY</button>
+    </header>
 
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
+    <!-- GAME WINDOW -->
```

Comment:
I like the fact that you add COMMENT when starting a new section, which makes it much easier to read your code. Nice!!!